commit 4b47b75833568258db62fc279764fb9f4c48b018 (refs/changes/41/3241/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-01-22T20:38:57+00:00 (1 year, 9 months ago)
    
    HEAD-2385 dhcpd config should use auto.ADMIN_IP instead of dhcpd_admin_ip
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/sapi_manifests/dhcpd/template b/sapi_manifests/dhcpd/template
index 3cd9c67..bc86c39 100644
--- a/sapi_manifests/dhcpd/template
+++ b/sapi_manifests/dhcpd/template
@@ -29,7 +29,7 @@
   "port": 67,
   "tftpRoot": "/tftpboot",
   "defaultGateway": "{{{compute_node_default_gateway}}}",
-  "serverIp": "{{{dhcpd_admin_ip}}}",
+  "serverIp": "{{{auto.ADMIN_IP}}}",
   "leaseTime": {{{dhcp_lease_time}}},
   "resolvers": {{{resolvers}}}{{^resolvers}}[]{{/resolvers}},
   "netmask": "{{{admin_netmask}}}",
